docs/api/dashboard.md

```diff
@@ -27,3 +27,4 @@ import ApiIntro from '../reuse/api-intro.md';
 |JP        |https://api.jp.sumologic.com/docs/#tag/dashboardManagement   |
 |US1       |https://api.sumologic.com/docs/#tag/dashboardManagement      |
 |US2       |https://api.us2.sumologic.com/docs/#tag/dashboardManagement  |
+|KR       |https://api.kr.sumologic.com/docs/#tag/dashboardManagementv1  |
```

Comment:
AFAICT in all these lists the intent is for the lists to be sorted alphabetically. I think it's worth sticking to it.

Reply:
Ah, I didn't even notice that! Good call.